src/mb_enhanced_cover_art_uploads/providers/base.ts

```diff
@@ -43,6 +46,15 @@ export abstract class CoverArtProvider {
      */
     abstract findImages(url: URL, onlyFront: boolean): Promise<CoverArt[]>
 
+    /**
+     * Postprocess the fetched images. By default, does nothing, however,
+     * subclasses can override this to e.g. filter out or merge images after
+     * they've been fetched.
+     */
+    postprocessImages(images: Array<[CoverArt, FetchedImage]>): Promise<FetchedImage[]> {
+        return Promise.resolve(images.map((res) => res[1]));
```

Comment:
```suggestion
    postprocessImages(images: FetchedImage[]): Promise<FetchedImage[]> {
        return Promise.resolve(images);
```
No longer in use since https://github.com/ROpdebee/mb-userscripts/pull/195/commits/6495e35fd9fd2cfcdcbfe2b96c9072703b24268a, I'm not even sure if it has ever been necessary to have access to the `CoverArt` object for that code to work (since `FetchedImage` also has the `comment` property which was used in the now removed code).
Simplifying this paramter also requires changes in all modules where this method is called or overwritten.

Reply:
Good catch!